core/domain/topic_services.py

```diff
@@ -828,6 +828,21 @@ def _are_nodes_valid_for_publishing(
         if node.id == story.story_contents.initial_node_id:
             _are_nodes_valid_for_publishing([node])
 
+    chapters_change_list = []
```

Comment:
Shouldn't you gate this with "if serial chapter launch flag is switched off"? (And maybe add a comment)

Reply:
Done!!!